README.md

```diff
@@ -34,7 +34,312 @@ This README documents the build, customisation and testing of these runtime imag
 
 **Do you want to learn more about using Node.js actions to build serverless applications?** Please see the main project documentation [here](https://github.com/apache/openwhisk/blob/master/docs/actions-nodejs.md) for that information.
 
-## Usage
+## Build Runtimes
+
+### You have 2 options to build the NodeJS runtime:
+- Building locally
+- Using OpenWhisk Actions.
+### This README walks you through how to do both
+
+# Building NodeJS Runtime Locally
+
+### Pre-requisites
+- [Docker](https://www.docker.com/)
+- [curl](https://curl.se/), [wget](https://www.gnu.org/software/wget/), or [Postman](https://www.postman.com/)
+
+
+0. Choose/create a folder of your liking and make sure docker daemon is running
+
+1. Clone this repo:
+```
+git clone https://github.com/apache/openwhisk-runtime-nodejs
+cd openwhisk-runtime-nodejs
+```
+
+1.1 Choose a NodeJS version. All build files reside inside `core/nodejsActionBase`. If you take a look into `core/nodejsActionBase/Dockerfile` you’ll see a line that looks like:
+```
+FROM node:lts-stretch
+```
+This will use the latest NodeJS version. But we want to be more specific. Now if you look into each of the Dockerfile’s of `core/nodejs14Action`, `core/nodejs12Action`, `core/nodejs10Action`, you’ll notice different nodeJS versions. Let’s go ahead with the 14 version. All you have to do is substitute the line above from `core/nodejsActionBase/Dockerfile` with the equivalent line from `core/nodejs14Action/Dockerfile` that looks like:
+```
+FROM node:14.16.0-stretch
+```
+
+Or in the command line you can simply type:
+```
+cp core/nodejs14Action/Dockerfile core/nodejsActionBase/
```

Comment:
Really do not like overwriting the Dockerfile in nodejsActionBase; is there an alternative (and one that we can make sure is part of `/gitignore` to avoid developers accidentally committing to the repo.?